src/components/ScholarshipsMadeSimpleSection.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Container } from '@material-ui/core';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import PropTypes from 'prop-types';
+import ScholarshipsMadeSimpleGrid from './ScholarshipsMadeSimpleContent';
+
+function TabPanel(props) {
+  const { children, value, index } = props;
+
+  return (
+    <div role="tabpanel" hidden={value !== index}>
+      {value === index && (
+        <Box p={3}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+export default function ScholarshipsMadeSimpleSection() {
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  return (
+    <Container style={{ background: 'theme.palette.background.paper' }}>
```

Comment:
I suggest using the [useStyles hook API](https://material-ui.com/styles/basics/#hook-api) instead.

Reply:
Ok got it on there